mlmodel/format/NeuralNetwork.proto

```diff
@@ -23,25 +22,37 @@
  * and a layer may not have input or output names that
  * refer to layers that are not yet defined.
  *
- *
  * For CoreML specification version <=3,
  * all inputs are mapped to static rank 5 tensors, with axis notations
  * [Sequence, Batch, Channel, Height, Width].
  *
  * From specification version 4 onwards (iOS >= 13, macOS >= 10.15), more options are available
- * (see enums "NeuralNetworkMultiArrayShapeMapping", "NeuralNetworkImageShapeMapping")
+ * (see enums ``NeuralNetworkMultiArrayShapeMapping``, ``NeuralNetworkImageShapeMapping``)
  * to map inputs to generic N-Dimensional (or N rank) tensors, where N >= 1.
  *
  * Each layer type may have specific constraints on the ranks of its inputs and outputs.
  *
+ * Several layers are available in 3 different variations, with the names ending
+ * in identifiers: like, static and dynamic. For instance, ``FillLike``,
```

Comment:
like --> ``like``
static ---> ``static`` etc

Reply:
Also please move this section below to the end of this bigger introduction.

Reply:
Done. Can you take a look again?